modules/misunderstood/src/backend/db.ts

```diff
@@ -104,17 +104,17 @@ export default class Db {
         .andWhere('id', '<=', messageId)
         .orderBy('id', 'desc')
         .limit(3)
-        .select('id', 'event'),
+        .select('id', 'event', 'createdOn'),
```

Comment:
why not orderBy createdOn instead of id ?

Reply:
I did not try it, that's a good point. Will do and report back